ui/admin/app/controllers/scopes/scope/workers/worker/tags.js

```diff
@@ -70,4 +113,16 @@ export default class ScopesScopeWorkersWorkerTagsController extends Controller {
     this.modalTag = apiTag;
     this.removalConfirmation = null;
   }
+
+  /**
+   * Opens the edit tag modal.
```

Comment:
should prob say toggles the edit tag model since the method does both, thoughts?

Reply:
It absolutely should. Fixing now.